core/arch/arm/plat-stm32mp1/main.c

```diff
@@ -355,3 +360,40 @@ unsigned int stm32_get_gpio_bank_clock(unsigned int bank)
 	assert(bank <= GPIO_BANK_K);
 	return GPIOA + bank;
 }
+
+#ifdef CFG_STM32_RNG
```

Comment:
Minor: the `ifdef` could be avoided by moving this function to its own file -- it doesn't seem to depend on static functions etc. But you can leave it here if you prefer.

Reply:
I think it is a good idea. This main.c looks a misc.c many platform specific stuff and it is a bit messy I think.
Thanks for the suggestion, I will do.